foia_hub/templates/about.html

```diff
@@ -20,6 +20,7 @@
             </p>
 
             <h2>Features</h2>
+            <p></p>
```

Comment:
I don't know but I think that's not how that should be laid out. @vz3 can clarify.